### PR TITLE
Replace USB cable link in Bluetooth doc

### PR DESCRIPTION
**Pull request by TomBrien**
## Proposed change

The current link is **not** a USB extension cable, it's a USB plug to plug cable for linking computers and no use in this case (without a further adapter). 

Note: I've had to use an EU link to Amazon as I couldn't find an appropriate cable (with ferrite) on the US store.
Sidenote: The ferrite really shouldn't be needed (or do anything) on a correctly manufactured cable with the shield grounded at the plug end. I would actually advocate for listing just a normal USB 3 extension.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bdraco** [comment]
It looks like the male to female one is out of stock at Amazon so it's showing the male to male one.  Looks like we may have caused a run on them

**Comment by bdraco** [comment]
Similar pr here 

https://github.com/home-assistant/home-assistant.io/pull/23633

**Comment by TomBrien** [comment]
We should probably just ditch the ferrite recommendation. The one pictured is going to be for the power lines only biased on the size. Best case the shield is getting broken and the data lines pulled out and bypass it, worst case they're going through it. A good (high) quality cable will probably have some PCB signal ferrites or other filters on board.

**Comment by bdraco** [comment]
High quality shielded cable should be just fine.  I picked the original one because its what I had for testing and worked well.

Ideally we link to a source that won't go away or out of stock

**Comment by TomBrien** [comment]
Totally agree. USB cables are a quality minefield (power only micro usb I'm looking at you!). Maybe best to just link to a name-brand cable?

**Comment by bdraco** [comment]
> Totally agree. USB cables are a quality minefield (power only micro usb I'm looking at you!). Maybe best to just link to a name-brand cable?

Maybe provide 3 examples so if one stops working they can figure it out ?

**Review by janick — COMMENTED** [review]
PR #23633 provides a better fix.

**Comment by TomBrien** [comment]
> Maybe provide 3 examples so if one stops working they can figure it out ?

I'd favour a single product link to keep things simple but if that's not going to be reliable I guess a few (that don't failover to something else) is ok. None, of course, is also an option.

**Comment by bdraco** [comment]
I think we can drop the link.

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -39,7 +39,7 @@ For Bluetooth to function on Linux systems, the [D-Bus](https://en.wikipedia.org
 
 Some systems may not come with Bluetooth and require a USB adapter. Installing an adapter for the first time may require multiple restarts for the device to be fully recognized.
 
-If you experience an unreliable Bluetooth connection, installing a [USB extension cable with a Ferrite Bead](https://a.co/d/fRnFIwy) may improve reliability.
+If you experience an unreliable Bluetooth connection, installing a [USB extension cable with a Ferrite Bead](https://amzn.eu/d/gZFaY7F) may improve reliability.
```
Comment (TomBrien) [inline source/_integrations/bluetooth.markdown]:
```suggestion
If you experience an unreliable Bluetooth connection, installing a short USB extension cable between your Bluetooth adapter and your Home Assistant server may improve reliability.
```

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks @TomBrien

**Pull request merged**